annotation/data_model.py

```diff
@@ -0,0 +1,216 @@
+from datetime import datetime, timezone
+from typing import Dict, Any, List
+import json
+import os
+import requests
+
+ODS_TYPE = "ods:type"
+AT_TYPE = "@type"
+ODS_ID = "ods:ID"
+AT_ID = "@id"
+MAS_ID = os.environ.get('MAS_ID')
+MAS_NAME = os.environ.get('MAS_NAME')
+
+
+def timestamp_now() -> str:
+    """
+    Create a timestamp in the correct format
+    :return: The timestamp as a string
+    """
+    timestamp = str(
+        datetime.now(tz=timezone.utc).strftime("%Y-%m-%dT%H:%M:%S.%f"))
+    timestamp_cleaned = timestamp[:-3]
+    timestamp_timezone = timestamp_cleaned + 'Z'
+    return timestamp_timezone
+
+
+def mark_job_as_running(job_id: str):
+    """
+    Calls DiSSCo's RUNNING endpoint to inform the system that the message has
+    been received by the MAS. Doing so will update the status of the job to
+    "RUNNING" for any observing users.
+    :param job_id: the job id from the kafka message
+    """
+    query_string = f"{os.environ.get('RUNNING_ENDPOINT')}/{MAS_ID}/{job_id}/running"
+    requests.get(query_string)
+
+
+def build_agent() -> Dict[str, Any]:
+    """
+    Builds agent based on MAS ID and name
+    :return: Agent object
+    """
+    return {
+        AT_ID: f"https://hdl.handle.net/{MAS_ID}",
+        AT_TYPE: 'as:Application',
+        'schema:name': MAS_NAME,
+        'ods:hasIdentifier': [
```

Comment:
Ah so far I often only used the hasIdentifier if there were additional identifiers and not for the first one. But this might actually be better.

Reply:
? What kind of identifiers are allowed here?

Reply:
All identifiers we can gather. So the @id in the object should be like a primary identifier, often generally orcid, wikidata, or a handle. Any other identifiers can go into this array. But adding the handle to the array might be smart as well.